core/field_checkbox.js

```diff
@@ -116,6 +126,14 @@ Blockly.FieldCheckbox.prototype.CURSOR = 'default';
  */
 Blockly.FieldCheckbox.prototype.isDirty_ = false;
 
+/**
+ * Character for the check mark. Used to apply a different check mark
+ * character to individual fields.
+ * @type {?string}
+ * @private
+ */
+Blockly.FieldCheckbox.prototype.checkChar_ = null;
```

Comment:
Moving forward we are no longer setting fields on the prototype as per the Google JS style guide: https://google.github.io/styleguide/jsguide.html#features-classes-fields
Let's move this to the constructor.

Reply:
Ok I moved it but I'm not sure if I did it correctly. Questions:

* Initialize to undefined? or null?
* @private first? or @type first?
* I removed the property description, is that correct?

Reply:
- Initialize to null.
- Type then private. 
- Keep property description, use multiline jsdoc rather than single line jsdoc format.

Reply:
Done!